src/test/java/pageobjects/SalesforceObjects/QuoteObject.java

```diff
@@ -6,16 +6,24 @@ public class QuoteObject {
 
     //Creating New Quotes
     public static By clickOnQuotes =By.xpath("//SPAN[@class='slds-truncate'][text()='Quotes']");
+    public static By clickOnQuotesTab =By.xpath("//div[@class='slds-context-bar']//a[@title='Quotes']");
+
+    public static By rhsQuote=By.xpath("(//div[@class='slds-media__body slds-align-middle']/h2[@class='slds-card__header-title']/a/span[@title='Quotes'])[3]");
```

Comment:
try to avoid creating xpath with the use of index as they can create problems in the future